csm/conf/etc/csm/components.yaml

```diff
@@ -48,14 +48,14 @@ s3_server:
         - /opt/seagate/auth/resources/authserver.properties
         - /opt/seagate/s3/conf/s3config.yaml
         - /etc/sysconfig/s3server-*
-        - /var/log/seagate/s3/s3server*INFO*
+        - /var/log/cortx/s3/s3server*INFO*
```

Comment:
Are we still using components.yaml?

Reply:
We are not using it and the removal is taken up in another story.